metaworld/envs/mujoco/sawyer_xyz/sawyer_sweep.py

```diff
@@ -48,13 +48,13 @@ def __init__(self, random_init=False):
             np.array(obj_low),
             np.array(obj_high),
         )
-        self.goal_space = Box(np.array(goal_low), np.array(goal_high))
 
         self.observation_space = Box(
             np.hstack((self.hand_low, obj_low,)),
             np.hstack((self.hand_high, obj_high,)),
         )
 
+        self._last_rand_vec = self.obj_init_pos
```

Comment:
Is there something special about this environment here? Please leave a comment if you remember.